ezyrb/database.py

```diff
@@ -67,11 +67,15 @@ def __getitem__(self, val):
                         self.scaler_parameters, self.scaler_snapshots)
 
     def __len__(self):
+        """
+        This method returns the number of snapshots
```

Comment:
Please specify the returned type (see the documentation for the `snapshots` "method")
Don't forget the dot at the end of the sentences.